blocks/small-promo-block/features/small-promo/default.jsx

```diff
@@ -37,7 +48,7 @@ const SmallPromo = ({ customFields }) => {
 			// does not need embed_html because no video section
 			// does not need website section nor label because no overline
 			// does not need byline because no byline shown
-			filter: `{
+			filters: `{
```

Comment:
This now seems to be pulling back a ton more information where just filter seemed to be more specific and related to the promo...

filter: 
![image](https://user-images.githubusercontent.com/2287238/203143834-9f981cef-d9f2-49cb-a698-cb679b1b5bc7.png)

filters;
![image](https://user-images.githubusercontent.com/2287238/203143981-b5e3df79-a207-4bf5-9a76-270714a407e5.png)

Reply:
I assume this was a test of some sort?

Reply:
Fixed